scripts/download.pl

```diff
@@ -70,27 +70,50 @@ ()
 	return undef;
 }
 
-sub download_cmd($) {
+sub download_cmd {
 	my $url = shift;
 	my $have_curl = 0;
-
-	if (open CURL, "curl --version 2>/dev/null |") {
+	my $have_aria2c = 0;
+	my $fn = shift;
+	my @mrs = @_;
```

Comment:
```suggestion
	my @additional_mirrors = @_;
```

Reply:
Also if we really want we can reduce this to `my $additional_mirrors =join(" ", map "$_/$filename", @_);` and drop the for later... 

And in the aria2c download function just use

`qw(aria2c --stderr -c -x2 -s10 -j10 -k1M), $url, $additional_mirrors ,`